appveyor.yml

```diff
@@ -126,8 +126,7 @@ before_deploy:
   - "SET PATH=%ORIGPATH%"
   - "%CMD_IN_ENV% %WP_INSTDIR%/scripts/env.bat"
   - "%CMD_IN_ENV% python.exe setup.py install"
-  - "%CMD_IN_ENV% easy_install --upgrade pip"
-  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui"
+  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui start_jupyter_cm"
```

Comment:
What I did to test was:
 - `pip install start_jupyter_cm`.
 - Check that the add executable runs from (WinPython) CMD.
 - Open up add executable and set a spurious path as shebang, now the add command fails.

 ```
> jupyter_context-menu_add.exe
Fatal error in launcher: Unable to create process using '"'
```
 - Change shebang to only `python.exe`, check that it runs correctly again.

This is with pip version 8.0.2.